### PR TITLE
Workaround of iDrac 9 with 04.40

### LINKED ISSUE #650: iDRAC firmware HTML5
Once verified, report HTML5  console required for 4.40

### PR DESCRIPTION
**Pull request by iranzo**
# Description

Workaround indication for 04.40

Closes #650 

## Type of change

Please select the appropiate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not being merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

**Comment by iranzo** [comment]
@iurygregory will this be ok from your PoV?

File: documentation/ipi-install/modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc [hunk 1]
```diff
@@ -13,18 +13,21 @@ The installer for installer-provisioned {product-title} clusters validates the h
...
+
 [NOTE]
 ====
 Refer to the hardware documentation for the nodes or contact the hardware vendor for information on updating the firmware.
 
 There are no known firmware limitations for HP servers.
 
-For Dell servers, ensure the {product-title} cluster nodes have AutoAttach Enabled through the iDRAC console. The menu path is: **Configuration->Virtual Media->Attach Mode->AutoAttach**. There is a known issue with iDRAC 9 firmware version 04.40.00.00.
+For Dell servers, ensure the {product-title} cluster nodes have AutoAttach Enabled through the iDRAC console. The menu path is: **Configuration->Virtual Media->Attach Mode->AutoAttach**. There is a known issue with iDRAC 9 firmware version 04.40.00.00 which has a workaround when setting the console to use HTML5 instead of the default setting.
 ====
```
Comment (iurygregory) [inline documentation/ipi-install/modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc:31]: I'm fine with the change, just wondering if we need to tell the path to change the console type (like we have in the AutoAttach)

Comment (iranzo) [reply documentation/ipi-install/modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc:31]: If you have it at hand I can add  to the PR to be validated by @johnwilkins

Comment (iurygregory) [reply documentation/ipi-install/modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc:31]: 1- login in the idrac page
2- access virtual console settings
3- On Virtual Console tab change the `Plug-in Type` to `HTML5`

Comment (johnwilkins) [reply documentation/ipi-install/modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc:31]: I sent in another upstream PR already on this. I will add the menu path to the downstream PR so we can get it in before COB today.

**Pull request closed without merging**